docs/book/application-integration/stand-alone.md

```diff
@@ -0,0 +1,214 @@
+# Stand-Alone
+
+The view and all view-helpers of laminas-view can also be used stand-alone.
+
+## The View
+
+The examples uses the following directory structure:
+
+* `public`
+    * `index.php`
+* `templates`
+    * `index.phtml`
+    * `layout.phtml`
+
+### Basic Example
+
+#### Setup
+
+[Create a renderer, set a resolver for templates](../php-renderer.md#usage)
+and initialize the view in `public/index.php`:
+
+```php
+// Create template resolver
+$templateResolver = new Laminas\View\Resolver\TemplatePathStack([
+    'script_paths' => [__DIR__ . '/../templates'],
+]);
+
+// Create the renderer
+$renderer = new Laminas\View\Renderer\PhpRenderer();
+$renderer->setResolver($templateResolver);
+
+// Initialize the view
+$view = new Laminas\View\View();
+$view->getEventManager()->attach(
+    Laminas\View\ViewEvent::EVENT_RENDERER,
+    static function () use ($renderer) {
+        return $renderer;
+    }
+);
+```
+
+#### Create View Script
+
+[Create a view script](../view-scripts.md) in `templates/index.phtml`:
+
+```php
```

Comment:
should it be php+html, not sure if it is supported

Reply:
This results in wrong classes: `+html language-php`